testing/php7-apcu/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
+# Maintainer: Carlo Landmeter <clandmeter@gmail.com>
+pkgname=php7-apcu
+_pkgreal=apcu
```

Comment:
That's strange suggestion because `_pkgreal` & `_pkgver` looks a standard all over php* packages